src/style.css

```diff
@@ -1 +1,74 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  font-weight: 300;
+  src:
+    url(../src/fonts/Avenir-Book.ttf) format('truetype'),
+    url(../src/fonts/Avenir-Heavy.ttf) format('truetype');
+}
+
+.body {
+  margin: 0;
+  box-sizing: border-box;
+  padding-top: 20px;
+  font-family: Avenir, Arial, sans-serif;
```

Comment:
It's good practice to include a fallback font family in case the main one fails to load. Adding a generic font family such as 'sans-serif' can ensure text is displayed with an acceptable font even if 'Avenir' is not available.